proto/streamlit/proto/TextInput.proto

```diff
@@ -27,6 +27,6 @@ message TextInput {
 
   string id = 1;
   string label = 2;
-  string default = 3;
+  string value = 3;
```

Comment:
Like Button.proto comment: To change the protobuf, choose a new numerical value for "value" and explicitly deprecate the word "default" as well as its number.